packages/medusa/src/services/cart.ts

```diff
@@ -1410,41 +1410,70 @@ class CartService extends TransactionBaseService {
    * Throws if discount regions does not include the cart region
    * @param cart - the cart to update
    * @param discountCode - the discount code
-   * @return the result of the update operation
    */
   async applyDiscount(cart: Cart, discountCode: string): Promise<void> {
+    return await this.applyDiscounts(cart, discountCode)
+  }
+
+  /**
+   * Updates the cart's discounts.
+   * If discount besides free shipping is already applied, this
+   * will be overwritten
+   * Throws if discount regions does not include the cart region
+   * @param cart - the cart to update
+   * @param discountCodes - the discount code(s) to apply
+   */
+  async applyDiscounts(
+    cart: Cart,
+    discountCodes: string | string[]
```

Comment:
suggestion: accept only string[] and pass in an array from line 1415. i.e `return await this.applyDiscounts(cart, [discountCode])`